site/docs/components/cookie_compliance.mdx

```diff
@@ -0,0 +1,136 @@
+
+import { Playground } from "docz";
+import { StatusLabel, FileInput } from "hds-react";
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+import Link from "../../src/components/Link";
+
+# Cookie compliance
+<StatusLabel type="alert">Draft</StatusLabel>
+
+<LargeParagraph>
+  The cookie compliance component informs users about cookie usage. This banner is shown when they visit a website or an application for the first time.
+</LargeParagraph>
+
+
+## Principles
+
+### Cookie clauses in English (EN)
+
+#### Banner text (accordion closed)
+**Headline:** [Service name] uses cookies
+
+**Bodycopy:** This website uses essential cookies to ensure site performance and for basic analytics. In addition, we use uses targeting cookies to improve the user experience, for analytics and showing targeted content. By continuing to use the site, you agree to the use of essential cookies.
+
+#### Banner text (accordion open)
+**Headline:** [Service name] uses cookies
+
+**Bodycopy:** This website uses essential cookies to ensure site performance and for basic analytics. In addition, we use uses targeting cookies to improve the user experience, for analytics and showing targeted content. By continuing to use the site, you agree to the use of essential cookies.
+
+**Subhead:** Information about the cookies used on the website
+
+**Bodycopy:** The cookies used on this website are classified according to their purpose. You can find information about each category below and allow or deny the use of cookies.
+
+**H:** Mandatory cookies
+
+**B:** The use of mandatory cookies cannot be denied. They enable the operation of the site and affect E.g. the usability of the site.
+(Checkbox + duration + information about the cookie)
+
+**H:** Optional cookies
+
+**B:** You can accept or decline optional cookies.
+(Checkbox + duration + information about the cookie)
+
+### UI components
+
+#### 1. Language selection
+The default UI componentes and cookie descriptions are delivered in Finnish, Swedish and English.
+
+#### 2. Cookie selection
+The cookies are divided to mandatory and optional cookies. The mandatory cookies are displayed "grayed-out" to visually support the fact that they cannot be unselected.
+
+#### Default checkboxes for cookie consent
+
+**Mandatory cookies** (visually grayed-out / disabled)
+
+- Login cookies: A mandatory cookie that allows the visitor to visit the site.
+- Language cookies: We store the interface language you have selected.
+
+**Optional cookies**
+
+- Statistic cookies: The information collected by statistical cookies is used to develop the website further.
+- Preference cookiess: Preference cookies customize E.g. the look and feel of a site based on a user's choices.
+- Marketing cookies: Marketing cookies can be applied to target specific content to users.
+
+#### 3. Consent confirmation buttons
+- Accept all cookies
+- Only mandatory
+- Continue with selected (TBD)
+- Opt out (TBD)
+- Close (cross icon) -button
+
```

Comment:
If I understand correctly, when there are optional consents and the banner is shown,  the buttons are "Accept all cookies" | "Only mandatory"

And when user opens the settings tab/accorion the buttons should change to: "Continue with selected" | "Only mandatory"

Reply:
Yes. That's my 5 cents as well.